2015-02-27  Jakub Jelinek  <jakub@redhat.com>

	PR tree-optimization/65048
	* gcc.dg/tree-ssa/ssa-dom-thread-9.c: Add -std=gnu89 to dg-options.
	(foo): Use K&R style definition.

--- gcc/testsuite/gcc.dg/tree-ssa/ssa-dom-thread-9.c	(revision 221062)
+++ gcc/testsuite/gcc.dg/tree-ssa/ssa-dom-thread-9.c	(revision 221063)
@@ -1,12 +1,13 @@
 /* PR 65048 */
 /* { dg-do compile } */
-/* { dg-options "-O3" } */
+/* { dg-options "-O3 -std=gnu89" } */
 
 int a, b, c, d;
 void fn (void);
 
 int
-foo (int x)
+foo (x)
+     int x;
 {
   switch (x)
     {
